egs/wsj/s5/utils/remove_symbols_from_phones.pl

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env perl
+# Copyright 2019 Junbo Zhang
```

Comment:
Sorry, I don't want this script to be in utils/, I think it should be in local/.  The issue is that in addition to the `_{BEIS}` markers, which are standard, it removes the stress markers, which are specific to CMUDict.

Reply:
Yes, indeed.  I have modified it. @danpovey